src/main/java/com/nucleus/chargepolicy/service/ChargePolicyServiceImpl.java

```diff
@@ -0,0 +1,57 @@
+package com.nucleus.chargepolicy.service;
+
+import com.nucleus.chargepolicy.dao.ChargePolicyDao;
+import com.nucleus.chargepolicy.model.ChargePolicy;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.ArrayList;
+import java.util.List;
+@Service
+public class ChargePolicyServiceImpl implements ChargePolicyService {
+
+    @Autowired
+    private ChargePolicyDao chargePolicyDao;
+    public ChargePolicyDao getChargePolicyDao() {
+        return chargePolicyDao;
+    }
+
+    public void setChargePolicyDao(ChargePolicyDao chargePolicyDao) {
+        this.chargePolicyDao = chargePolicyDao;
+    }
+
+   /* public void getChargePolicyList(ChargePolicy chargePolicy) {
+        System.out.println("In Service");
+        this.chargePolicyDao.getChargePolicyList();
+    }*/
+
+
+
+    public void setEligibilityPolicyDAO(ChargePolicyDao chargePolicyDao) {
+        this.chargePolicyDao = chargePolicyDao;
+    }
+    public void insert(ChargePolicy chargePolicy) {
+        System.out.println("In Service");
+        this.chargePolicyDao.insert(chargePolicy);
+    }
+    public List<String> getChargeCodes(){
```

Comment:
why is this method there, if its not getting anything from the Dao? is it used anywhere else?

Reply:
I'll leave this here incase there is some dependency of this somewhere